model/clusters_mgmt/v1/version_type.model

```diff
@@ -37,6 +37,10 @@ class Version {
 	// HostedControlPlaneEnabled indicates whether this version can be used to create HCP clusters.
 	HostedControlPlaneEnabled Boolean
 
+	// HostedControlPlaneDefault is a flag that indicates if this should be selected as the default version when a
+	// HCP cluster is created without specifying explicitly the version.
```

Comment:
Only thing I'm not sure is how the model generator behaves for 2 lines of comments. Haven't seen it before

Reply:
@gdbranco Should be okay, the [EndOfLifeTimestamp](https://github.com/openshift-online/ocm-api-model/blob/084e74c4f3a0378e757e8d30d5418bd0b9c19e79/model/clusters_mgmt/v1/version_type.model#L48) also has two line of comments.